### DISCUSSION TITLE
Error 2147500037, Send Later Do not Work

### DISCUSSION THREAD

**Discussion by alenunezok** [Extended-Thunder/send-later#302] [Q&A]
I got a problem using send later. When y put send in to 5 minutes, after 5 minutes thunderbird don't send the email and show a error call 2147500037. It's says "Error copying scheduled message to Outbox for delivery (code 2147500037)".  I verified the problem en extended git hub and I did everything that the page indicated. But it does not work and i do not understand why. Someone could help me?

**Comment by jph-sendlater** [comment]
What version of Thunderbird are you using? What version of Send Later do you have installed?

**Comment by jikamens** [comment]
Closed due to lack of response. Also, this is probably either a corrupt Thunderbird profile or a full temporary directory.
